README.md

```diff
@@ -19,3 +13,30 @@ Provided rules:
 
 To test a `pybind_extension`, the most common approach is to write the test in
 python and use the standard `py_test` build rule.
+
+## Installation
+
+In your workspace:
+
+```starlark
+git_repository(
```

Comment:
Perhaps switch this to `http_archive`? From [the Bazel documentation](https://docs.bazel.build/versions/2.0.0/external.html#repository-rules):

> Prefer http_archive to git_repository and new_git_repository.

GitHub provides a mechanism to fetch a commit as an archive, so the `url` and `strip-prefix` would be:


```
  strip-prefix = "pybind11_bazel-<stable-commit>",
  urls = ["https://github.com/pybind/pybind11_bazel/archive/<stable-commit>.zip"],
```

Reply:
Cool, I didn't know this was still a thing.